packages/network-controller/tests/provider-api-tests/shared-tests.ts

```diff
@@ -106,48 +116,51 @@ export const testsForProviderType = (providerType: ProviderType) => {
 
     describe('methods that assume there is no block param', () => {
       const assumingNoBlockParam = [
-        // { name: 'eth_getFilterLogs', numberOfParameters: 1 },
-        // { name: 'eth_blockNumber', numberOfParameters: 0 },
```

Comment:
Just wanted to confirm the reason for this change. I found [a comment thread on the PR were this was introduced](https://github.com/MetaMask/core/pull/1087#discussion_r1102164822) that suggests this method acted roughly the same as on the extension, but was more difficult to test because the block tracker uses this method internally and calls it an indeterminate number of times.

If that's the case, then this should be fine to exclude from our list of differences. Does that sound correct?

Reply:
yep, they should behave the same